packages/react-core/src/components/ClipboardCopy/ClipboardCopyToggle.tsx

```diff
@@ -27,7 +27,7 @@ export const ClipboardCopyToggle: React.FunctionComponent<ClipboardCopyTogglePro
     onClick={onClick}
     id={id}
     aria-labelledby={`${id} ${textId}`}
-    aria-controls={contentId}
+    aria-controls={isExpanded ? contentId : undefined}
```

Comment:
This update needs to be made as it resolves a critical axe error where the aria-controls attr references a non-existent ID (since the expandable content is dynamically rendered)